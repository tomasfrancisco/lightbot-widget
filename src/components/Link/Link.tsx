import { darken } from "polished";
import styled from "styled-components";

export const Link = styled<any, "a">("a")`
  color: ${props => props.theme.linkColor};

  &:visited {
    color: ${props => props.theme.linkColor};
  }

  &:active {
    color: ${props => darken(0.2, props.theme.linkColor ? props.theme.linkColor : "#4c4c4c")};
  }
`;
