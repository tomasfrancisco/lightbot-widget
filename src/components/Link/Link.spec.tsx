import { Link } from "components/Link/Link";
import { shallow } from "enzyme";
import * as React from "react";

describe("modules/Widget/Window", () => {
  let component;

  beforeEach(() => {
    component = shallow(<Link />);
  });

  it("renders", () => {
    expect(component).toMatchSnapshot();
  });
});
