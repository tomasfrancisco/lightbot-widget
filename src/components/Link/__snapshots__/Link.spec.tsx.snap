// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`modules/Widget/Window renders 1`] = `
<a
  className="sc-bdVaJa hRUuKy"
/>
`;
